rinkeby-deployment.md

```diff
@@ -58,7 +58,7 @@ Repeat passphrase:
 "0xbf4696ecfa2d3697f98805d4166fdaeaf3b67944"
 ```
 
-Your private key is stored in a secret facility (`~/Library/Ethereum/rinkeby/keystore`). If you wish to view your account on [myetherwallet](https://www.myetherwallet.com/#view-wallet-info) just select the option of `Keystore File (UTC / JSON)` and point it to the file at this address. Enter your passphrase and you should see your account.
+Your private key is stored in a secret facility (_Linux/OS X:_ `~/Library/Ethereum/rinkeby/keystore`, _Windows:_  `%userprofile%\AppData\Roaming\Ethereum\rinkeby\keystore`). If you wish to view your account on [myetherwallet](https://www.myetherwallet.com/#view-wallet-info) just select the option of `Keystore File (UTC / JSON)` and point it to the file at this address. Enter your passphrase and you should see your account.
```

Comment:
I'm pretty sure this is only valid for MacOS and not Linux :)

Reply:
Oops :)
Don't have a Linux system on hand, I'm guessing there it ends up in in `~/.ethereum/...` ?

Reply:
Don't know that :D just leave it blank until someone with a linux system comes along ;-)